### PR TITLE
GCWeb Implementation Doc: Fix Broken Links

### PR DESCRIPTION
**Pull request by BrahimMahadi**

### CODE REVIEW

File: docs/implementing.html [hunk 1]
```diff
@@ -207,13 +207,13 @@ <h2 id="static-header-footer">Static header and footer</h2>
 </tr>
 <tr>
 	<th>Header</th>
-	<td><a href="https://github.com/wet-boew/GCWeb/blob/master/site/pages/gcweb-theme/static-header-footer/bootstrap-3.html#L1-L77" rel="external">Source code, line 1 to 77</a></td>
-	<td><a href="https://github.com/wet-boew/GCWeb/blob/master/site/pages/gcweb-theme/static-header-footer/bootstrap-4.html#L1-L66" rel="external">Source code, line 1 to 66</a></td>
+	<td><a href="https://github.com/wet-boew/GCWeb/docs/static-header-footer/bootstrap-3.html#L1-L77" rel="external">Source code, line 1 to 77</a></td>
```
Comment (GormFrank) [inline docs/implementing.html:210]: Ends at line 86 now.
```suggestion
	<td><a href="https://github.com/wet-boew/GCWeb/blob/master/docs/static-header-footer/bootstrap-3.html#L1-L86" rel="external">Source code, line 1 to 86</a></td>
```
Similar for footer and for Bootstrap 4.

**Review by GormFrank — CHANGES_REQUESTED** [review]

File: docs/implementing.html [hunk 2]
```diff
@@ -236,7 +236,7 @@ <h2 id="static-header-footer">Static header and footer</h2>
 <tr>
 	<th>Special assets</th>
 	<td>All included in the current GCWeb release</td>
-	<td><a href="https://github.com/wet-boew/GCWeb/blob/master/site/pages/gcweb-theme/static-header-footer/site-assets/x-bootstrap-4.css" rel="external">Download experimental CSS</a></td>
+	<td><a href="https://github.com/wet-boew/GCWeb/docs/static-header-footer/site-assets/x-bootstrap-4.css" rel="external">Download experimental CSS</a></td>
```
Comment (Ricokola) [inline docs/implementing.html:239]: This is still a broken link, I suggest using the following url: https://github.com/wet-boew/GCWeb/blob/master/docs/static-header-footer/site-assets/x-bootstrap-4.css

**Review by Ricokola — CHANGES_REQUESTED** [review]
Please fix the broken link at line 239

File: docs/implementing.html (hunk 2 above)
Comment (BrahimMahadi) [reply docs/implementing.html]: The broken link at line 239 has been fixed

**Review by Ricokola — APPROVED** [review]

**Review by Ricokola — APPROVED** [review]
These changes have been completed as requested.

**Review by Ricokola — APPROVED** [review]
These changes have been completed as requested.

**Review by GormFrank — APPROVED** [review]

**Pull request merged**